<?xml version="1.0"?>
<launch>
<param name = "robot_name" value = "main_robot"/>
<param name = "ODOM_RATE" value = "40"/>
<param name = "STATUS_RATE" value = "25"/>
  <inclued file="$(find eurobot_stm)/config/stm.yaml" command="load"/>
	<node name="stm_node" pkg="eurobot_stm" type="STM.py" output="screen"/>
</launch>

